scripts/sync-to-service.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-console */
+const fetch = require('node-fetch');
+
+const env = process.env.ENV;
+
+const HOSTS = {
+  staging: 'https://data-dictionary.staging-service.newrelic.com',
+  production: 'https://data-dictionary.service.newrelic.com',
+  'eu-production': 'https://data-dictionary.service.eu.newrelic.com',
+};
+
+const sync = async () => {
```

Comment:
I'm assuming that any uncaught errors in this script will be reported as a failed action in GitHub?

Reply:
They should yes since it would exist with a non-zero exit code. I've run the script myself locally to test, just not through the GH action.